### PR TITLE
Update alexa.smart_home.markdown

### PR DESCRIPTION
**Pull request by andyb2000**
Clarification on Account Linking - no port numbers, it will cause a failure.

## Proposed change
Simple documentation change/clarification to help users.

## Type of change
Documentation addition of information.

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/alexa.smart_home.markdown [hunk 1]
```diff
@@ -227,7 +227,7 @@ Alexa needs to link your Amazon account to your Home Assistant account. Therefor
 - Find the skill you just created, click `Edit` link in the `Actions` column.
 - Click `ACCOUNT LINKING` in the left navigation bar of build page
 - Do not turn on the "Allow users to link their account to your skill from within your application or website" switch. This will require a Redirect URI, which won't work.
-- Input all information required. Assuming your Home Assistant can be accessed by `https://[YOUR HOME ASSISTANT URL]`
+- Input all information required. Assuming your Home Assistant can be accessed by `https://[YOUR HOME ASSISTANT URL]` (IMPORTANT - Alexa account linking will NOT work with a non-standard port, so do not include a :1234 or similar port number, it MUST be on port 443 (use your firewall to forward))
```
Comment (c0ffeeca7) [inline source/_integrations/alexa.smart_home.markdown]:
```suggestion
- Input all information required. Assuming your Home Assistant can be accessed by `https://[YOUR HOME ASSISTANT URL]`. Alexa account linking does not work with a non-standard port. You must use port 443. Use your firewall to forward, if needed. Using a `:1234` or a similar port number will not work.
```

**Review by c0ffeeca7 — COMMENTED** [review]
small style tweak

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @andyb2000 :+1: 
I do not have Alexa myself, will wait for feedback from @ochlocracy

**Comment by andyb2000** [comment]
> Thank you, @andyb2000 +1 I do not have Alexa myself, will wait for feedback from @ochlocracy

No problem, to confirm it was something I was carrying out today and hit this issue and confirmed this resolved it.

**Review by frenck — APPROVED** [review]
This is correct, thanks @andyb2000 👍 

../Frenck

**Pull request merged**